jit/v2/JIT_v2.0.0.yaml

```diff
@@ -366,22 +448,21 @@ paths:
               examples:
                 internalServerErrorExample:
                   summary: |
-                    Internal Server Error while processing Port Call Service
+                    Internal Server Error while processing Terminal Call
                   description: |
                     An Internal Server Error has occurred - the consumer should contact {provider-support} and provide the `providerCorrelationReference` (in the example this is `4426d965-0dd8-4005-8c63-dc68b01c4962`)
 
-                    **NB**: `errorCode` not yet standardized by DCSA. Value `7005` is just a "random example"
+                    **NB**: `errorCode` not yet standardized by DCSA. Value `7003` is just a "random example"
                   value:
                     httpMethod: PUT
-                    requestUri: /v2/port-call-services/085a3207-5e45-49cf-8e1b-f8442beaf545/estimated-timestamp
+                    requestUri: /v2/terminal-calls/085a3207-5e45-49cf-8e1b-f8442beaf545
                     statusCode: 500
                     statusCodeText: Internal Server Error
-                    statusCodeMessage: |
-                      Internal Server Error occurred while processing Port Call Service
+                    statusCodeMessage: Internal Server Error occurred while processing Port Call Service
```

Comment:
Terminal call?

Reply:
Fixed along with a lot of other places having the same error :-)